### ISSUE TITLE
Scenario Summary Results and Scenario Results Unable to be Open at Same Time

### ISSUE THREAD

**Issue by rnugent3** [HydrologicEngineeringCenter/HEC-FDA#891]
I attempted to open the results for a given scenario to compare to the scenario summary results to confirm that the results are the same - and hit an uncaught exception. I think that this is a reasonable workflow that we should ideally accommodate if practical, and if not practical, we should handle the exception at a minimum. 

![image](https://github.com/HydrologicEngineeringCenter/HEC-FDA/assets/25163165/69abaf42-138d-47df-b93c-9c9ba81d1519)

**Comment by codymccoy** [comment]
Dang, i hate that both oxyplot and scichart don't let you use the same plot model for multiple plots. Sounds like we just need to clone the model.

**Comment by rnugent3** [comment]
Not sure if relevant or related - after making this attempt, I am no longer able to view the results I attempted to review. I closed out of the summary results, right-clicked on the scenario again and selected view results, and nothing happens. I can access results for other scenarios.

**Comment by codymccoy** [comment]
I don't think this has anything to do with the summary results since those don't have any plots. I got the exception when trying to pop out the scenario results.
I tested popping out every UI that had a plot and found that the reguency editor and the ias results both throw this exception but the others do not.
![image](https://github.com/HydrologicEngineeringCenter/HEC-FDA/assets/47757848/8a787adc-8b88-4aa0-a50b-3869a0226c0d)
